koku/masu/processor/tasks.py

```diff
@@ -566,6 +536,7 @@ def update_summary_tables(  # noqa: C901
 
     # Apply OCP on Cloud tasks
     if signature_list:
+        LOG.info(log_json(tracing_id, "chaining deletes and summaries"), report)
```

Comment:
Is there an extra parenthesis here?
```suggestion
        LOG.info(log_json(tracing_id, "chaining deletes and summaries", report)
```

Reply:
Yea, there were a couple other places with this errant parenthesis. Good catch! I think I fixed them all.